### ISSUE TITLE
Sushi project should contain all initial project files

### ISSUE THREAD

**Issue by troelde** [HL7Norway/Consent#1]
_genonce.bat
_genonce.sh
_updatePublisher.bat
_updatePublisher.sh
.gitignore

I had to generate this myself by copying from another project.

**Comment by thomiz** [comment]
@troelde Why do you need these files? If you use HL7 [autobuilder](https://github.com/FHIR/auto-ig-builder/blob/master/README.md) or a [build script](https://github.com/HL7Norway/VkpObservation/blob/master/.github/workflows/vkpobservation-gh-pages.yml) triggered from GitHub, the files are not necessary at all.

**Comment by troelde** [comment]
@thomiz : Agree, but I am a newbie here, so who can configure the HL7 autobuilder or build script? 

At least you need the .gitignore file.

**Comment by thomiz** [comment]
I have never used the autobuilder actually, but the build script is easy enough to setup. I can probably setup a build script for you if you have working fsh code, ig.ini and sushi-config.yaml.

You are correct, the .gitignore is hightly recommended for all projects. The other files are probably a good idea if you want to be able to build it on you own computer using the HL7 publisher.

**Comment by troelde** [comment]
@thomiz :  I compiled fsh code and sushi reported no errors or warnings.

**Comment by thomiz** [comment]
@troelde Jeg har oppdatert prosjektet med en workflow. For å teste den må den pulles til main først. Jeg håper det virker ut av boksen, men kan ikke garantere det selvfølgelig siden dere har en litt annen katalogstruktur enn det jeg pleier å bruke, så jeg måtte editere litt på scriptet. Vi fikser det hvis det kommer feilmeldinger.

Ny byggeprosess som startes fra **Actions** skjermen. Scriptet blir ikke synlig før man har lagt .github/workflow katalogen til main.

Resultatet av bygget skal legges under branchen **gh-pages** og katalogen **currentbuild**

**Comment by thomiz** [comment]
Did a update of the writeup describing the build process, would apprecieate any feedback.

[build-pipeline](https://github.com/thomiz/build-pipeline)
